### ISSUE TITLE
Output of webhooks appear incorrectly in chat

### ISSUE THREAD

**Issue by luckenbach** [RocketChat/Rocket.Chat#11669]
### Description:

Formatting of attachments is appearing odd in desktop / web apps.

### Steps to reproduce:

1. Create a webhook
2. POST data to webhook

### Expected behavior:

Webook's fields should appears stacked and not horizontal

Phone appears normal
![image](https://user-images.githubusercontent.com/533852/43657991-0b91301c-971d-11e8-83b4-c456e2129789.png)

### Actual behavior:

![image](https://user-images.githubusercontent.com/533852/43657805-79582494-971c-11e8-833e-0a3bf93ed3d1.png)

### Server Setup Information:

- Version of Rocket.Chat Server:  0.68.1
- Operating System:  Linux/k8s
- Deployment Method: docker
- Number of Running Instances: 1
- DB Replicaset Oplog: Disabled
- NodeJS Version: v8.11.3
- MongoDB Version: 3.4.7

**Comment by kaiiiiiiiii** [comment]
For testing purposes, could you please provide the Webhook payload that is sent to the chat and your custom webhook script (if used)?

**Comment by Scream85** [comment]
I have another funny one related to webhooks, they appear in different room too, but there the message is: "This message was ignored", since running v0.68.x.

**Comment by luckenbach** [comment]
# Payload:
```{
    "schemaId": "Microsoft.Insights/activityLogs",
    "data": {
        "status": "Activated",
        "context": {
            "activityLog": {
                "authorization": {
                    "action": "Microsoft.Compute/virtualMachines/delete",
                    "scope": "/subscriptions/11/resourceGroups/kafka-staging/providers/Microsoft.Compute/virtualMachines/testmachine"
                },
                "channels": "Operation",
                "caller": "c",
                "correlationId": "f",
                "description": "",
                "eventSource": "Administrative",
                "eventTimestamp": "2018-07-31T18:10:04.8374492+00:00",
                "httpRequest": "{\"clientRequestId\":\"\",\"clientIpAddress\":\"123\",\"method\":\"DELETE\"}",
                "eventDataId": "123",
                "level": "Informational",
                "operationName": "Microsoft.Compute/virtualMachines/delete",
                "operationId": "123",
                "resourceId": "/subscriptions/123/resourceGroups/kafka-staging/providers/Microsoft.Compute/virtualMachines/testmachine",
                "resourceGroupName": "kafka-staging",
                "resourceProviderName": "Microsoft.Compute",
                "status": "Started",
                "subStatus": "",
                "subscriptionId": "123",
                "submissionTimestamp": "2018-07-31T18:10:23.1631897+00:00",
                "resourceType": "Microsoft.Compute/virtualMachines"
            }
        },
        "properties": {}
    }
}
```
# Webhook
```class Script {
  process_incoming_request({ request }) {
    // console is a global helper to improve debug
    console.log(request.content.data.context)


    var alertColor = "good";

    // if(request.content.status === "Resolved"){ alertColor = "good"; }
    // else if (request.content.status === "Activated") { alertColor = "danger"; }
    var ad = request.content.data.context.activityLog;

    return {
      content:{
        text: "Azure Activity Notification",
        attachments: [{
          title: ad.operationName,
          pretext: ad.description,
          color: alertColor,
          fields: [
            {
              title: "Resource",
              value: ad.resourceId + '\n'
            },
            {
              title: "Event Type",
              value: ad.eventSource
            },
            {
              title: "Level",
              value: ad.level
            },
           {
              title: "Operation",
              value: ad.operationName
            },
            {
              title: "Status",
              value: ad.status
            }
          ]
        }]
       }
    };

    return {
       error: {
         success: false,
         message: 'Error'
       }
    };
  }
}
```

**Comment by kaiiiiiiiii** [comment]
By default every attachment field is ‘short’ and allows another field to be right next to it. If you don’t want that behavior you can simply add ‘"short": false’ to a field. Adding that field parameter should do the magic for you :)

**Comment by luckenbach** [comment]
```short: True or False``` doesn't seem to effect this

**Comment by graywolf336** [comment]
This will be closed/fixed by #11740
